frontend/src/views/Transactions/Transactions.jsx

```diff
@@ -15,8 +15,8 @@ import { useTranslation } from 'react-i18next'
 import { Role } from '@/components/Role'
 import { transactionsColDefs } from './_schema'
 import { useCurrentUser } from '@/hooks/useCurrentUser'
+import { ORGANIZATION_STATUSES, TRANSACTION_STATUSES, TRANSFER_STATUSES } from '@/constants/statuses'
 import { roles, govRoles } from '@/constants/roles'
-import { ORGANIZATION_STATUSES, TRANSFER_STATUSES } from '@/constants/statuses'
 import OrganizationList from './components/OrganizationList'
```

Comment:
Due to slow connection, I tried to click more than once on "Recommend" button on the confirmation Modal window and it ended up creating multiple records.
<img width="850" alt="image" src="https://github.com/bcgov/lcfs/assets/130073308/921236f1-d6b0-48ff-b43e-e59c55519a37">

Reply:
I added a loading indicator to the BCButton and the Modal triggers it on button click now. This prevents the double clicking.